src/main/java/com/fortanix/sdkms/performance/sampler/AbstractEncryptionAndDecryptionSampler.java

```diff
@@ -54,6 +55,10 @@ public void setupTest(JavaSamplerContext context) {
         super.setupTest(context);
         this.securityObjectsApi = new SecurityObjectsApi(this.apiClient);
         SobjectRequest sobjectRequest = new SobjectRequest().name(UUID.randomUUID().toString()).objType(objectType).keySize(keySize);
+        if(CryptMode.fromValue(mode) == CryptMode.FPE) {
+            FpeOptions fpeOptions = new FpeOptions().radix(10).preserve(Arrays.asList(0,1,2,3,-1,-2,-3,-4)).luhnCheck(true).name("Credit Card");
```

Comment:
Please add a comment mentioning that currently we are supporting only credit card numbers for FPE encryption.

Reply:
Taken care of.